src/pinning.js

```diff
@@ -133,7 +133,7 @@ class Pinning {
 
   _onMessage (topic, data) {
     console.log(topic, data)
-    if (!data.type || data.type === 'PIN_DB') {
+    if (data.type === 'PIN_DB' && OrbitDB.isValidAddress(data.odbAddress)) {
```

Comment:
anything changed to remove `!data.type`? did this case handle messages with no type as well

Reply:
It was basically a compatibility hack for an old version the 3box lib that didn't have type. No one should be using that now anyway :)